src/set_theory/cardinal/basic.lean

```diff
@@ -1363,6 +1363,9 @@ lemma mk_subtype_mono {p q : α → Prop} (h : ∀x, p x → q x) : #{x // p x}
 lemma mk_union_le_omega {α} {P Q : set α} : #((P ∪ Q : set α)) ≤ ω ↔ #P ≤ ω ∧ #Q ≤ ω :=
 by simp
 
+@[simp] lemma mk_subset (t : set α) : #{s // s ⊆ t} = 2 ^ #t :=
```

Comment:
Just to reiterate my previous comment for other reviewers now that the context is clear; I'd prefer to state this as 
```suggestion
@[simp] lemma mk_subset (t : set α) : #{s // s ⊆ t} = #(set t) :=
```
but will let another reviewer tie break

Reply:
And to reiterate, I very strongly disagree, for two reasons:
- `cardinal.mk_set` is a `simp` lemma, so any application of `cardinal.mk_subset` on `simp` would be accompanied by `cardinal.mk_set` anyways.
- There's many more lemmas on cardinal exponentiation than on the cardinality of sets.

Reply:
I'm not really sure what the rationale you're giving here is, to be honest.

Reply:
It's moot anyway, this lemma already exists as `cardinal.mk_powerset`

Reply:
Oh, can't believe I missed that 🤦🏻‍♀️. The equivalence also exists already as `equiv.set.powerset`, so there's not really much point to this PR.